### DISCUSSION TITLE
Domain sender rewrite

### DISCUSSION THREAD

**Discussion by Rez0k** [postalserver/postal#3255] [Help with using Postal]
Hi,

Let's assume I have 2 authenticated domains, a.com and b com.
How can I send a message that the FROM field will be from a.com, but the domain sender will be b.com. (I think it's called spoofing).

The user will see in his mailbox a mail from user@a.com, but the mail will be sent from b.com.

Is this possible?

**Comment by willpower232** [comment]
if you enable the sender header in the settings in the UI you should be able to do this I believe

**Reply by Rez0k** [reply]
It's sending and adding on bahaf of, any idea how to remove this?

**Reply by willpower232** [reply]
I think it has come up here before, I can't remember if people are able to resolve it or not.
